lib/spack/external/_pytest/vendored_packages/pluggy.py

```diff
@@ -500,22 +500,10 @@ def check_pending(self):
     def load_setuptools_entrypoints(self, entrypoint_name):
```

Comment:
Like `_mark_plugins_for_rewrite`, I think this function should be removed entirely, and the only call to it (in `_preparse`) should be removed OR commented out with an explanation that we don't use any plugins.

Reply:
This is done, and actually simplifies the patching, as we only need to patch `config.py` to remove the only call.

Reply:
I think the whole `_mark_plugins_for_rewrite` function should be removed (or the function body should be replaced with `raise Exception...`) to avoid the possibility of reintroducing a dependency on Setuptools.